senza/arguments.py

```diff
@@ -0,0 +1,48 @@
+import click
+import re
+
+REGION_PATTERN = re.compile(r'^[a-z]{2}-[a-z]+-[0-9]$')
+
+
+def validate_region(ctx, param, value):
+    """Validate Click region param parameter."""
+    if value is not None:
+        if not REGION_PATTERN.match(value):
+            raise click.BadParameter("'{}'. Region must be a valid "
+                                     "AWS region.".format(value))
+    return value
+
+
+region_option = click.option('--region',
+                             envvar='AWS_DEFAULT_REGION',
+                             metavar='AWS_REGION_ID',
+                             help='AWS region ID (e.g. eu-west-1)',
+                             callback=validate_region)
+
+parameter_file_option = click.option('--parameter-file',
+                                     help='Config file for params',
+                                     metavar='PATH')
+
+output_option = click.option('-o', '--output',
+                             type=click.Choice(['text', 'json', 'tsv']),
+                             default='text',
+                             help='Use alternative output format')
+
+json_output_option = click.option('-o', '--output',
+                                  type=click.Choice(['json', 'yaml']),
+                                  default='json',
+                                  help='Use alternative output format')
+
+stacktrace_visible_option = click.option('--stacktrace-visible',
+                                         is_flag=True,
+                                         help='Show stack trace instead of '
+                                              'storing it')
```

Comment:
Maybe it is a good idea to move the code:
```python
HandleExceptions.stacktrace_visible = stacktrace_visible
```
to something like a `callback=set_stacktrace_visible` here?